content/en/real_user_monitoring/session_replay/mobile/privacy_options.md

```diff
@@ -200,6 +268,23 @@ On Android, users can select the `mask_large_only` setting, which replaces image
     )
     SessionReplay.enable(with: sessionReplayConfig)
 
+{{< /code-block >}}
+{{% /tab %}}
+{{% tab "React Native" %}}
+{{< code-block lang="typescript" filename="App.tsx" disable_copy="false" collapsible="true" >}}
```

Comment:
Hi @marco-saia-datadog, we currently describe how the "mask content images" specifically works for iOS and Android [here](https://docs-staging.datadoghq.com/marcosaia/RUM-7156/rn-session-replay-docs/real_user_monitoring/session_replay/mobile/privacy_options/?tab=reactnative#mask-content-images). Is there any detail we want to specifically include about how it works for React Native as well?

Reply:
React Native is delegating to the native sdk strategies under the hood. So if it runs on Android it should mask content images if they exceed certain dimensions, while on ios it will mask system images.